packages/clerk-js/src/ui/elements/FormControl.tsx

```diff
@@ -94,30 +94,41 @@ function useFormTextAnimation() {
   };
 }
 
+type CalculateConfigProps = {
+  recalculate?: string | boolean | undefined;
```

Comment:
How come we need boolean here ?

Reply:
@panteliselef we don't. I had the thought initially to make it a boolean value but later revised